Geometry/MTDCommonData/data/CrystalBar/mtdStructureTopology.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
```

Comment:
I'm missing something here. Is this a relative path?

Reply:
@vargasa this is the usual xml header of all the DD geometry files, referring to https://cmssdt.cern.ch/lxr/source/DetectorDescription/Schema/DDLSchema.xsd
For its structure @ianna can probably better comment, I guess it is historical